### PR TITLE
feat: :sparkles: support for adding script url added for umami

### PR DESCRIPTION
**Pull request by AkashRajpurohit**
## What does this PR do?

- Adds support for adding the script URL while setting up Umami App.
- The current version of the app only works with hosted solution but with this, users can use their self-hosted version of Umami as well to set up Analytics in cal.com (ref: https://github.com/calcom/cal.com/pull/14783#issuecomment-2104420069)

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ ] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). If N/A, write N/A here and check the checkbox.
- [ ] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Add the Umami App to your booking.
- Set Script URL along with Site ID.
- Verify the events coming in on your Umami app when the event page is opened.

SCR-20240821-l8q

SCR-20240821-l9e

SCR-20240821-l9w

## Checklist

- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: packages/app-store/umami/config.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "tag": {
       "scripts": [
         {
-          "src": "https://us.umami.is/script.js",
+          "src": "{SCRIPT_URL}",
```
Comment (anikdhabal) [inline packages/app-store/umami/config.json:17]: I think it's better to let the user choose whether they want to add SCRIPT_URL or not

**Review by anikdhabal — CHANGES_REQUESTED** [review]

File: packages/app-store/umami/config.json (hunk 1 above)
Comment (anikdhabal) [reply packages/app-store/umami/config.json:17]: Looks good

**Review by anikdhabal — APPROVED** [review]
LGTM!

**Comment by avocadoattack** [comment]
I've been following this PR carefully, and I'm wondering when the Umami Analytics integration will be available within Cal

**Pull request merged**